### PR TITLE
[MRG] DOC: better theme?

### PR DESCRIPTION
**Pull request by jasmainak**
This looks a bit better to my eye than the orange theme. Any opinions @dylansdaniels @cjayb @ntolley @rythorpe ?

image

here are a list of themes: https://bootswatch.com/

### CODE REVIEW

**Comment by dylansdaniels** [comment]
@jasmainak definitely better! Should I look into customizing the colors to match our HNN theme? 

We use the following:
BLUE: HEX #1156A3
PURPLE: HEX # 474095 
VIOLET: HEX #812A89 
YELLOW: HEX #e5a734

**Comment by jasmainak** [comment]
@dylansdaniels do you want to give it a shot? We can merge this and then you can iterate on top of that.

Basically you'd have to add a CSS file according to the instructions here:
https://sphinx-bootstrap-theme.readthedocs.io/en/stable/README.html#adding-custom-css

You can preview the website by clicking on "Details" here:
image

or by running:

```sh
$ cd doc/
$ make html
```

or (faster ... does not run examples):

```sh
$ make html-noplot
```

**Comment by cjayb** [comment]
Is that 'Flatly'? Anyway, LGTM, black is the new orange.

**Comment by jasmainak** [comment]
@dylansdaniels feel free to merge!

**Comment by jasmainak** [comment]
this is "yeti" theme @cjayb

**Comment by ntolley** [comment]
@jasmainak appologies I was a bit hasty on merging #357. Do you think you could commit a `whats_new.rst` update here? Unless you want to wait until it's a proper public method...

**Comment by jasmainak** [comment]
> @jasmainak appologies I was a bit hasty on merging #357.

no worries, it's good that you are merging pull requests :)

> Unless you want to wait until it's a proper public method...

I would wait until it's a proper public method :) I noticed some issues with the whats_new document though, I'll make a separate pull request for that.

**Comment by jasmainak** [comment]
I pushed the changes for separating the "tutorials" and "how tos" in this pull request. Let me know what you think.

@stephanie-r-jones this might be a good moment to try opening the Circle CI build and go to artifacts, check the new rendering of the docs and offer any comments you may have. I have mostly just reorganized things but @dylansdaniels can build upon this

**Comment by jasmainak** [comment]
this would be good to merge sooner than later @cjayb @ntolley as it touches many files

File: examples/howto/README.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+How to
+------
+
+These are short examples that show how to use HNN-core API to accomplish something
```
Comment (cjayb) [inline examples/howto/README.txt:4]:
```suggestion
These are short examples that show how to use HNN-core API to accomplish specific
tasks.
```

**Review by cjayb — APPROVED** [review]
I have just to one totally insignificant suggestion, other LGTM for at merge. Please do the honours @ntolley

**Review by ntolley — APPROVED** [review]
Love the new theme! Committed the suggested change and will merge when checks are green.

**Comment by jasmainak** [comment]
Thanks @ntolley for pushing the fix!

**Pull request merged**